### PR TITLE
Update references to repo name

### LINKED ISSUE #337: Should `cardano-chain` be renamed to `cardano-ledger` or something similar?
The specs of `cardano-chain` reside in `fm-ledger-rules`. Should this repo be renamed to something like `cardano-ledger` (since it actually implement the ledger rules).

cc: @ruhatch @nc6 

Feel free to close if the answer is no :)

### PR DESCRIPTION
**Pull request by ruhatch**
Closes #337

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +61,7 @@ particular needs of the work being done. Such file would be a copy of the
 
 ## Formatting
 
-This repo uses `brittany` to enforce a consistent formatting style.
+This repo uses `brittany` to encourage a consistent formatting style.
```
Comment (dnadales) [inline README.md:64]: :smile:

**Review by dnadales — APPROVED** [review]

**Comment by dnadales** [comment]
CI failing on 

```
stack --dump-logs --color always build --fast --no-run-tests --bench --no-run-benchmarks --haddock --haddock-internal --no-haddock-deps
cabal file path /build/cardano-chain/test/cardano-chain-test.cabal does not match the package name it defines.
Please rename the file to: cardano-ledger-test.cabal
For more information, see: https://github.com/commercialhaskell/stack/issues/317
```

File: bors.toml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 status = [
 
   # Buildkite: stack build
-  "buildkite/cardano-chain",
+  "buildkite/cardano-ledger",
   # Hydra: build for required jobset
-  "ci/hydra:serokell:cardano-chain:required",
+  "ci/hydra:serokell:cardano-ledger:required",
```
Comment (jbgi) [inline bors.toml]: should now be `Cardano` instead of `serokell`.

**Pull request merged**